docs/en/docs/tutorial/index.md

```diff
@@ -2,15 +2,13 @@
 
 This tutorial shows you how to use **FastAPI** with most of its features, step by step.
 
-Each section gradually builds on the previous ones, but it's structured to separate topics, so that you can go directly to any specific one to solve your specific API needs.
+Each section gradually builds on the previous ones. Each section is a separate topic, so that you can go directly to any specific topic to solve your specific API need.
```

Comment:
I actually prefer the original version as it doesn't need repeating "each section".

Reply:
Good point.  I think there are some ways to improve the sentence. *It's* is ambiguous and *structured to separate topics* doesn't sound quite right. Additionally, the sentence would be easier to understand if it were broken up.

How about this phrasing:
```
Each section gradually builds on the previous ones and  is structured into separate topics. You can go directly to any section to solve your specific API needs